### ISSUE TITLE
Translate new backend field

### ISSUE THREAD

**Issue by Tay987** [oc-shopaholic/oc-orders-shopaholic-plugin#113]
Hi,
I'm trying to extend the plugin adding new text field to description tab of category section of the backend (where I can create new category or update existing one), the field appear correctly and its value is saved in database.
The only thing I couldn't do is to make it translatable (the en/it switcher on the input field does not appear).
I cannot understand what I am doing wrong.
This is the class I have created following the plugin documentation.

<pre>
class ExtendCategoryModel
{
    public function subscribe()
    {
        'Category::extend(function ($obCategory) {
            /** @var Category $obCategory */

            $obCategory->fillable[] = 'title';
            $obCategory->translatable[] = 'title';
            
            $obCategory->addCachedField(['title']);
        });
    }
}
</pre>

**Comment by kharanenka** [comment]
Hi! What field type do you have for 'title'?

**Comment by kharanenka** [comment]
Do you have en/it switcher on 'name' field?
Have you verified that event code is being executed?
You have syntax error:
Your example:
```php
'Category::extend(function ($obCategory) {
```
Corrected version:
```php
Category::extend(function ($obCategory) {
```

**Comment by Tay987** [comment]
In my code there isn't the syntax error, it was a typo opening the issue, sorry.

_Do you have en/it switcher on 'name' field?_
Yes I have.

_Have you verified that event code is being executed?_
How can I do it? If the event is not executed, I shouldn't see the text field on the form, correct?

**Comment by kharanenka** [comment]
No, ```Category::extend``` extends Category model only, not backend form.

**Comment by Tay987** [comment]
Ok, how I can check if my event is executed?

**Comment by kharanenka** [comment]
```php
public function subscribe()
{
dd('test');
//....
}
```

**Comment by Tay987** [comment]
I confirm the event is executed.

**Comment by kharanenka** [comment]
Perhaps your events fire later than events of Translite plugin. Try to change priority for event that expands backend form.

**Comment by Tay987** [comment]
how?

**Comment by kharanenka** [comment]
Please send your code where you add 'title' field to backend.

**Comment by Tay987** [comment]
<pre>
class ExtendCategoryFieldsHandler extends AbstractBackendFieldHandler
{

    /**
     * Extend field
     * @param \Backend\Widgets\Form $obWidget
     */
    protected function extendFields($obWidget)
    {
        $arAdditionFields = [
            'title' => [
                'label'   => 'lovata.toolbox::lang.field.title',
                'tab'     => 'lovata.toolbox::lang.tab.description',
                'type'    => 'text',
                'span'    => 'full'
            ],
        ];

        $obWidget->addTabFields($arAdditionFields);
    }

    /**
     * Get model class name
     * @return string
     */
    protected function getModelClass() : string
    {
        return Category::class;
    }

    /**
     * Get controller class name
     * @return string
     */
    protected function getControllerClass() : string
    {
        return CategoryController::class;
    }
}
</pre>

while in my Plugin.php
<pre>
public function boot()
    {
        Event::subscribe(ExtendCategoryFieldsHandler::class);
        Event::subscribe(ExtendCategoryModel::class);
    }
</pre>

**Comment by kharanenka** [comment]
Try to add 'title' field to translatable array in Category model file

**Comment by Tay987** [comment]
I tried, but nothing changes.

**Comment by kharanenka** [comment]
Try to add  'title' field config to Category model .yml config file
